Newsletters: add 278 (2023-11-22)

harding opened the pull request:
- [x] Lede, update releases/RCs, add topic links @harding 
- [x] Client/Services section @bitschmidty 

Note: we're planning to do the spaces recap on Wednesday (22 Nov) instead of the usual Thursday due to U.S. thanksgiving.  (Maybe we want to publish the newsletter on Tuesday?  I'm ok with that.)

bitschmidty commented:
Id prefer the Weds publication consistency over the additional time between news and pod personally.

bitschmidty reviewed (commented):
lgtm so far

vostrnad commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,123 @@
+
+<!-- FIXME:harding to update Tuesday -->
+
+- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
+  version of the predominant full node implementation. There's a [testing
+  guide][26.0 testing] and a scheduled
+  meeting of the [Bitcoin Core PR Review Club][] dedicated to testing on
+  15 November 2023.
```
The testing meeting has already happened:
```suggestion
- [Bitcoin Core 26.0rc2][] is a release candidate for the next major
  version of the predominant full node implementation. A [testing
  guide][26.0 testing] is available.
```

vostrnad commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,123 @@
+  channel identifier_ (SCID), which only uses 8 bytes. Because channels
+  are shared by two nodes, an additional bit is prepended to the SCID to
+  specifically identify one of the two nodes.  Because offers may often
+  be used in size-constrained media, this space savings is significant.
```
```suggestion
  be used in size-constrained media, the space savings are significant.
```

t-bast reviewed (commented):
LGTM for the `Offers-compatible LN addresses` section, thanks for this nice summary!

harding commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,176 @@
+  channels, defaults to [bech32m][topic bech32] receive addresses, and adds
+  additional [zero conf channel][topic zero-conf channels] support.
+
+- **Durabit whitepaper announced:**
```
Regarding the two papers in this section, I think we should keep them for this newsletter, but I would prefer in the future that we encourage the authors of ideas to post them to one of our [sources](https://bitcoinops.org/internal/sources/).  We want ideas like these to have a chance to be peer reviewed before we write about them.

bitschmidty replied:
I think this is fair

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,183 @@
+      address of the spender and the receiver.
+
+    - _Risk of theft:_ the server operator can man-in-the-middle
+      invoices in order to steal funds.
```
```suggestion
      invoices to steal funds.
```

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,183 @@
+      records that directly store offers for particular addresses.  For
+      example, a DNS `TXT` record, `alice._lnaddress.domain.com`,
+      includes an offer for Alice.  Another record,
+      `bob._lnaddress.domain.com`includes an offer for Bob.  Teinturier
```
```suggestion
      `bob._lnaddress.domain.com` includes an offer for Bob.  Teinturier
```

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,183 @@
+      per user (and update that record if the user needs to change their
+      default offer).
+
+  As of this writing, there had not been any public discussion of the
```
```suggestion
  As of this writing, there has not been any public discussion of the
```

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,183 @@
+  The [BitStream whitepaper][] and [early prototype][bitstream github] layout a
+  protocol for the hosting and atomic exchange of digital content for
+  coins using timelocks and merkle trees with verification and fraud
+  proofs.  For previous discusison of paid data transfer protocols, see
```
```suggestion
  proofs.  For previous discussion of paid data transfer protocols, see
```

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```diff
@@ -0,0 +1,183 @@
+- **Durabit whitepaper announced:**
+  The [Durabit whitepaper][] outlines a protocol using [timelocked][topic
+  timelocks] Bitcoin transactions in conjunction with a chaumian-style mint
+  to incentivize seeding of large files.
```
```suggestion
  to incentivize the seeding of large files.
```

LarryRuane commented on _posts/en/newsletters/2023-11-22-newsletter.md:
```suggestion
  version of the predominant full-node implementation. There's a [testing
```

vostrnad replied:
This doesn't look right. A grep through the `_posts/en` directory gives 240 results for "full node" and only 6 for "full-node", all of which are a part of a URL.

harding replied:
I don't have a strong preference either way.  I suggest we leave it as-is in the merged newsletter and I'll try to remember this discussion the next time someone suggests the hyphenated / single-adjective version and we can debate it then.

harding commented:
Made edits for all feedback (thanks @vostrnad @bitschmidty @LarryRuane @t-bast !), added lede, updated releases/RCs, and added topic entries.  Reviewed client/services section (thanks @bitschmidty !)  Thanks everyone!

bitschmidty commented:
Reviewed topic entries, lede, and latest review comments. Squashed and merged, thank you all!
